### DISCUSSION TITLE
Deno (1.28 +) now supports npm mdoules, what about next 13 support for deno?

### DISCUSSION THREAD

**Discussion by shafqatalix** [vercel/next.js#42901] [Ideas]
### Describe the feature you'd like to request

Deno (1.28) now support npm packages, what about using next 13 with deno?

### Describe the solution you'd like

Next13 with deno (instead of NodeJS)

### Describe alternatives you've considered

Deno

**Comment by raulmarindev** [comment]
https://deno.com/blog/v1.28

**Comment by orzklv** [comment]
https://deno.com/blog/v1.30
